test/integration/dom.css.classes.test.ts

```diff
@@ -91,6 +98,20 @@ describe('Bpmn Elements registry - CSS class management', () => {
       bpmnVisualization.bpmnElementsRegistry.removeCssClasses(['lane_01', 'userTask_0'], ['class1', 'class3']);
       htmlElementLookup.expectLane('lane_01', { label: 'Lane 3', additionalClasses: ['class2'] });
       htmlElementLookup.expectUserTask('userTask_0', { label: 'User Task 0', additionalClasses: ['class2'] });
+
+      // remove several classes by passing classes in a single string separated with spaces
+      // this is not working (remember this is working for 'add css classes')
+      bpmnVisualization.bpmnElementsRegistry.addCssClasses(['userTask_0'], ['extra-class1', 'extra-class2', 'extra-class3']);
+      htmlElementLookup.expectUserTask('userTask_0', { additionalClasses: ['class2', 'extra-class1', 'extra-class2', 'extra-class3'] });
+      bpmnVisualization.bpmnElementsRegistry.removeCssClasses(['userTask_0'], ['extra-class1 extra-class2 extra-class3']);
+      htmlElementLookup.expectUserTask('userTask_0', { additionalClasses: ['class2', 'extra-class1', 'extra-class2', 'extra-class3'] });
```

Comment:
I don't understand why the removing of the extra classes in a single string separated with spaces is not working.

The adding working in https://github.com/process-analytics/bpmn-visualization-js/pull/2267/files#diff-4869fee480545e4cb7ed19e6229000a8bf2b2c55afc3da1f06c8c024dfdf4e66R46.
I'll expect to have the same behavior on the removing.

Reply:
To be clear, the tests just show the current behavior and not how the code should behave.
We discover this behavior by accident while working on some demos.

The discrepancies from the add and remove/toggle come from the way we store the classes in the internal CSS registry.
In the integration tests, we check what is written in the DOM, but when doing CSS classes add/remove/toggle we first pass through the CSS registry.

In the CSS registry, when we add "class-1 class2", it stores the string as it is i.e. a single string, not two. The SVG group class attribute contains this single string, but as space is the separator, it is like we pass 2 strings.
However, we cannot remove a 'class-1': when we request a remove, as there is no entry for this name in the registry, the class list remains unchanged.

Reply:
I did see it was the current behavior.

I'm ok to merge the PR Like this, and fix the behavior in a next PR 🙂 
Otherwise, good catch 👍🏻